lib/node_modules/@stdlib/ndarray/base/some-by/benchmark/benchmark.1d_rowmajor.js

```diff
@@ -0,0 +1,138 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var isBoolean = require( '@stdlib/assert/is-boolean' ).isPrimitive;
+var pow = require( '@stdlib/math/base/special/pow' );
+var scalar2ndarray = require( '@stdlib/ndarray/from-scalar' );
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
+var shape2strides = require( '@stdlib/ndarray/base/shape2strides' );
+var pkg = require( './../package.json' ).name;
+var someBy = require( './../lib' );
+
+
+// VARIABLES //
+
+var types = [ 'float64' ];
+var order = 'row-major';
+
+
+// FUNCTIONS //
+
+/**
+* Predicate function.
+*
+* @param {*} value - ndarray element
+* @returns {boolean} result
+*/
+function clbk( value ) {
+	return value > 0.0;
+}
+
+/**
+* Creates a benchmark function.
+*
+* @private
+* @param {PositiveInteger} len - ndarray length
+* @param {NonNegativeIntegerArray} shape - ndarray shape
+* @param {string} xtype - ndarray data type
+* @returns {Function} benchmark function
+*/
+function createBenchmark( len, shape, xtype ) {
+	var x;
+	var n;
+
+	x = discreteUniform( len, 1, 100 );
+	x = {
+		'dtype': xtype,
+		'data': x,
+		'shape': shape,
+		'strides': shape2strides( shape, order ),
+		'offset': 0,
+		'order': order
+	};
+	n = scalar2ndarray( 5.0, {
```

Comment:
```suggestion
	n = scalar2ndarray( 5, {
```

This same change should be made throughout this PR.

Reply:
There seems to be confusion as to when and why we add decimal values. We use decimal values to signal in our documentation when a value can take any real value. That is not the case here. `n` should never be, e.g., `5.5`, as that doesn't make sense. So, leave out the decimal in order to convey that only integer values should be provided.